Project/DAQ_System/Program/Src/Sensor/GyroScope/l3gd20h.hpp

```diff
@@ -0,0 +1,62 @@
+/*
+* Gyroscope L3GD20H
+* Authors:       William Lim
+*                Alex Shi
+* Modified By:   Cristian Cruz
+*
+* Email:         wll180000@utdallas.edu
+*                ahs190001@utdallas.edu
+*                cris14.alex@gmail.com
+*
+* (c) 2023 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#ifndef GYROSCOPE_L3GD20H_H
+#define GYROSCOPE_L3GD20H_H
+
+#include "igyroscope.hpp"
+#include <stdint.h>
+
+#define L3GD20_CTRL_REG1     0x20
+#define L3GD20_OUT_X_L       0x28
+#define GYR_ADDRESS          0xD6
+
+
+
+class Gyroscope_L3GD20H: public IGyroscope{
```

Comment:
Rename the class from `Gyroscope_L3GD20H` to simply `L3GD20H`. The team had mutually decided to shorten the name as it is already reflected that this is a gyroscope through the folder directory, the interface class, and later in our docs.